src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -30,7 +30,7 @@
       </v-col>
 
       <!-- Actions -->
-      <v-col cols="2" class="mt-n2" v-if="!hasNameTranslationChange && !isSummaryMode">
+      <v-col cols="2" class="mt-n2" v-if="isHideEditNameTranslationButton">
```

Comment:
Maybe this property should be renamed :)

Reply:
Shoot, you're right, I reversed it when it was correct before